apps/crn-server/test/data-providers/users.data-provider.test.ts

```diff
@@ -567,11 +567,11 @@ describe('User data provider', () => {
       await userDataProvider.fetch(fetchOptions);
 
       const expectedFilter =
-        "data/onboarded/iv eq true and data/role/iv ne 'Hidden' and" +
-        " ((contains(data/firstName/iv, '%22')" +
-        " or contains(data/lastName/iv, '%22')" +
-        " or contains(data/institution/iv, '%22')" +
-        " or contains(data/expertiseAndResourceTags/iv, '%22')))";
+        "data/onboarded/iv eq true and not(data/role/iv eq 'Hidden') and" +
+        " ((contains(data/firstName/iv,'\"'))" +
```

Comment:
as you can see the test says `Should sanitise double quotation mark by encoding to hex` - clearly after your changes it no longer encodes to hex but escapes it with a backslash instead. I would not only make sure to change the tests - i would actually make sure that this still works as normal in squidex